app/workers/publishing_api_document_republishing_worker.rb

```diff
@@ -16,21 +16,51 @@ def perform(document_id)
     document = Document.find(document_id)
     published_edition = document.published_edition
     pre_publication_edition = document.pre_publication_edition
+    latest_edition = document.latest_edition
 
+    if edition_has_been_unpublished?(latest_edition)
+      send_drafts_and_unpublish(latest_edition)
+    else
+      send_published_and_pre_publication(published_edition, pre_publication_edition)
+    end
+  end
+
+private
+
+  def edition_has_been_unpublished?(edition)
+    edition.unpublishing.present?
```

Comment:
That state is a bug as described in [Trello](https://trello.com/c/zYKOV4us/476-unpublishings-are-not-deleted-when-the-unpublished-content-is-republished-medium). We need to fix that and run the data migration from https://github.com/alphagov/whitehall/pull/2706